resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -323,6 +339,16 @@ class VisualEffect
             {
                 $transparencyValue = if ($this.TransparencyEffects) { 0 } else { 1 }
 				
+                if (-not (DoesRegistryKeyPropertyExist -Path $global:PersonalizationRegistryPath -Name ([VisualEffect]::TransparencySettingProperty))) {
+                    New-ItemProperty -Path $global:PersonalizationRegistryPath -Name ([VisualEffect]::TransparencySettingProperty) -Value $transparencyValue -PropertyType DWord
+                }
+                Set-ItemProperty -Path $global:PersonalizationRegistryPath -Name ([VisualEffect]::TransparencySettingProperty) -Value $transparencyValue 

```

Comment:
You set `PersonalizationRegistryPath` here but you check `ControlPanelAccessibilityRegistryPath` above. Which one is the correct path?